### ISSUE TITLE
Add lateral links between tools with related features

### ISSUE THREAD

**Issue by linl33** [opendatakit/tool-suite-X#87]

**Comment by SondosAhmed** [comment]
hi 
Id like to try this issue 
but can you add more description to make it more accurate

**Comment by wbrunette** [comment]
@SondosAhmed, I believe this issue is to look at review our documentation and add links between different sections when features or a different tool our mentioned.

Perhaps @elmps2018 has some advice on this.

**Comment by elmps2018** [comment]
@SondosAhmed, @wbrunette has the right idea. There are a lot of tools that are related, so for example in the xlsxconverter documentation, when it discusses all the prompts, to note that to see examples of how these appear in survey you click here.

**Comment by josdebum** [comment]
Hi @elmps2018 can I work on this issue.

**Comment by elmps2018** [comment]
@josdebum absolutely! I assigned this issue to you! There's some suggestions on how to get started in previous comments -- reading and trying out the docs and then figuring out how things are inter-related and adding links is the goal!

**Comment by Goodiec** [comment]
@wbrunette, please I would like to work on this issue.

**Comment by cvemann** [comment]
> @wbrunette, please I would like to work on this issue.

I don't think this issue is related to the current May 2023 cohort

**Comment by Goodiec** [comment]
> > @wbrunette, please I would like to work on this issue.
> 
> I don't think this issue is related to the current May 2023 cohort

@wbrunette mentioned that we can also work on other issues in the ODK-X issue queue and that it would count as contribution to the project

**Comment by wbrunette** [comment]
@Goodiec I have assigned the issue to you.

**Comment by Giftemmanuel** [comment]
hello ,Ms @wbrunette  this is my first time contributing to an open source project through Outreachy.. could i please be assigned this task?

**Comment by elmps2018** [comment]
Hi @Giftemmanuel, someone else has already been assigned to this task (you can see in assignees), please try to find a different issue to work on.
